test/metabase/api/dashboard_test.clj

```diff
@@ -107,6 +107,7 @@
                  (dissoc :id)
                  (assoc :created_at (boolean created_at)
                         :updated_at (boolean updated_at))
+                 (update :last_viewed_at boolean)
```

Comment:
This is an old test pattern, the better way to do this sort of stuff now is to use `=?`